lib/projfs.c

```diff
@@ -1466,14 +1576,22 @@ int projfs_create_proj_dir(struct projfs *fs, const char *path, mode_t mode)
 	if (fd == -1)
 		return errno;
 
-	res = set_xattr_empty(fd, 1);
+	if (set_xattr_empty(fd, XATTR_CREATE) == -1) {
+		res = errno;
+		goto out_close;
+	}
 
+	res = iter_user_xattrs(fd, attrs, nattrs,
+			       PROJ_XATTR_WRITE | PROJ_XATTR_CREATE);
+
+out_close:
 	close(fd);
 	return res;
 }
 
 int projfs_create_proj_file(struct projfs *fs, const char *path, off_t size,
-                            mode_t mode)
+                            mode_t mode, struct projfs_attr *attrs,
+			    unsigned int nattrs)
```

Comment:
Spacing irregularity between this line and the one above!

Reply:
Indeed -- I didn't futz with the existing spacing but tried to "tabify" on the new line, but am more than happy to do the futzing since I'm here.  :-)

(Just FWIW, I've got a big pile of these kinds of tweaks in ce392b422b but am holding off until most of the functionality and test work is in place.)

Reply:
👍 👍 👍  Fair!!